gp-populate-anything/gppa-ignore-current-user-filter-for-admins.php

```diff
@@ -26,6 +26,9 @@
 	}
 
 	array_pop( $query_builder_args['where'][ $filter_group_index ] );
+	if ( count( $query_builder_args['where'][ $filter_group_index ] ) == 0 ) {
+		$query_builder_args['where'] = array();
+	}
```

Comment:
What if there are multiple groups?

For example, if `$filter_group_index` is `1` here, and we have another on `2`, this would clear that out as well.